client/profile-wizard/steps/usage-modal.js

```diff
@@ -94,64 +87,70 @@ class UsageModal extends Component {
 			return null;
 		}
 
-		const { allowTracking } = this.state;
 		const { isRequesting } = this.props;
-		const trackingMessage = interpolateComponents( {
-			mixedString: __(
-				'Get improved features and faster fixes by sharing non-sensitive data via {{link}}usage tracking{{/link}} ' +
-					'that shows us how WooCommerce is used. No personal data is tracked or stored.',
-				'woocommerce-admin'
-			),
-			components: {
-				link: (
-					<Link
-						href="https://woocommerce.com/usage-tracking"
-						target="_blank"
-						type="external"
-					/>
+		const { isRequestStarted } = this.state;
+		const isBusy = isRequestStarted && isRequesting;
+
+		const title =
+			this.props.title ||
```

Comment:
Optional, but we could use default props for these to clean up the logic here a bit.

Reply:
Hm, I'm concerned about translations. If I understand correctly default props values will be evaluated at the import time and will be used in place of empty props. While current solution evaluates fallbacks at render time. What do you think?